modules/tasks/tasks/src/main/resources/webapp/js/directives.js

```diff
@@ -831,6 +831,7 @@
                         changeYear: true,
                         dateFormat: 'yy-mm-dd',
                         timeFormat: 'HH:mm z',
+                        yearRange: '-100y:c+nn',
```

Comment:
can we also leave 10 future years available (2016-2026 currently), since I believe we can choose that currently.